### PR TITLE
Event ref delete

### PR DESCRIPTION
**Pull request by foxj17**
- added the Ereza custom crash package [link](https://github.com/Ereza/CustomActivityOnCrash)
- delete deleted event's from user's attend/maybe list in DB upon viewing an event
- added a custom 404 page

### CODE REVIEW

File: MKESocial/app/src/main/res/layout/activity_not_found.xml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        android:textAppearance="@style/TextAppearance.AppCompat.Title"
+        android:textColor="@android:color/holo_red_dark"
+        tools:layout_editor_absoluteX="45dp"
+        tools:layout_editor_absoluteY="168dp" />
+
+    <TextView
+        android:id="@+id/textView2"
+        android:layout_width="215dp"
+        android:layout_height="106dp"
+        android:text="Sorry, looks like the pae you are looking for doesn't exist anymore"
```
Comment (TheDS) [inline MKESocial/app/src/main/res/layout/activity_not_found.xml:24]: pae -> page

**Review by TheDS — COMMENTED** [review]
Looks good, but is the error page intended to be used yet? Because as of right now, a crash results in the default page with the lady bug as shown in your link.

File: MKESocial/app/src/main/res/layout/activity_not_found.xml (hunk 1 above)
Comment (foxj17) [reply MKESocial/app/src/main/res/layout/activity_not_found.xml:24]: 😃 thanks for catching that!
and pushing up the crash results to do a custom page rn

**Review by ongda — DISMISSED** [review]
I didn't check anything fyi for all

**Review by jamesonn — DISMISSED** [review]
me neither

**Review by TheDS — APPROVED** [review]

**Pull request merged**